src/MenuStyle.php

```diff
@@ -150,23 +160,25 @@ class MenuStyle
      * @var array
      */
     private static $defaultStyleValues = [
```

Comment:
Could you maybe revert the whitespace changes here please? I prefer to keep them not aligned, it helps with diffs.